doc/INSTALL.html

```diff
@@ -160,7 +160,7 @@
 master branch are available at
 <A HREF="http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html.">http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html.</A>
 If you wish to build another version of QGIS, ensure to checkout the appropriate
-release branch.
+release branch. QGIS source files are currently available at https://github.com/qgis/QGIS.
```

Comment:
"currently" - is it planned to change this?

Reply:
You might have missed the blog post. Let me find it for you... 😄 

More seriously, not that I know. But in case...
Should I remove it? (*no strong opinion, myself*)

Reply:
> You might have missed the blog post. Let me find it for you...

:worried: jeez!

I'd probably make it a link `The QGIS source code can be found [in the repository https://github.com/qgis/QGIS]`.